config/config.go

```diff
@@ -62,7 +60,7 @@ func (c *Config) EnableTLS() bool {
 	if c.SSLConfig.Acme != nil {
 		return true
 	}
-	return c.SSLConfig.Key != "" || c.SSLConfig.Cert != ""
+	return c.SSLConfig.Key != "" && c.SSLConfig.Cert != ""
```

Comment:
This is *technically* a breaking change, but I don't see how anyone can use SSL with only cert or only key. Both are required for it to work.

Reply:
Yep, correct, this is a bug